doc/sphinx/dev_checklist.rst

```diff
@@ -51,26 +53,26 @@ The environment variables most relevant for a POD's operation are:
 
    1. Output figures to ``$WK_DIR/obs/`` and ``$WK_DIR/model/`` respectively.
 
-   2. ``$WK_DIR/obs/PS`` and ``$WK_DIR/model/PS``: If a POD chooses to save vector-format figures, save them as ``EPS`` under these two directories. Files in these locations will be converted by the framework to ``PNG`` for HTML output. Caution: avoid using ``PS`` because of potential bugs in recent ``matplotlib`` and converting to PNG.
+   2. ``$WK_DIR/obs/PS/`` and ``$WK_DIR/model/PS/``: If a POD chooses to save vector-format figures, save them as ``EPS`` under these two directories. Files in these locations will be converted by the framework to ``PNG`` for HTML output. Caution: avoid using ``PS`` because of potential bugs in recent ``matplotlib`` and converting to PNG.
 
-   3. ``$WK_DIR/obs/netCDF`` and ``$WK_DIR/model/netCDF``: If a POD chooses to save any digested data for later analysis/plotting, save them in two directories in ``NetCDF``.
+   3. ``$WK_DIR/obs/netCDF/`` and ``$WK_DIR/model/netCDF/``: If a POD chooses to save digested data for later analysis/plotting, save them in these two directories in ``NetCDF``.
```

Comment:
@jkrasting @yihungkuo @tsjackson-noaa Why would we continue to support PS graphics in upcoming versions if they cause known issues in vital plotting libraries?

Reply:
There's been interest in "publication quality" output which has traditionally been postscript format.  Personally, I prefer high-res (>300dpi) PNG format.

Reply:
Is EPS alone insufficient for the requirement for publication-quality vector images? The documentation states that only PS causes a problem.

Reply:
@wrongkindofdoctor I don't think this is still a problem. Matplotlib has been updated to support eps format since the start of phase 2.
@jkrasting High-res PNG would be fine. But if a POD produces many figures, the overall size of the output may become a problem.

Reply:
@yihungkuo Thanks for clarifying. It seems like eliminating PS support in the future (Version 4?) would be a safe bet. Note that I'm all for supporting multiple figure formats if most developers require it. I don't think it is a good idea to retain unnecessary features that are known to break things (even if we include disclaimers in the docs).

Reply:
I'm OK with targeting this for depreciation in version 4.